modules/CmpdReg/src/server/routes/CmpdRegRoutes.coffee

```diff
@@ -32,7 +32,6 @@ exports.setupRoutes = (app, loginRoutes) ->
 	app.post '/cmpdReg/metalots', loginRoutes.ensureAuthenticated, exports.metaLots
 	app.post '/cmpdReg/salts', loginRoutes.ensureAuthenticated, exports.saveSalts
 	app.post '/cmpdReg/isotopes', loginRoutes.ensureAuthenticated, exports.saveIsotopes
-	app.post '/cmpdReg/api/v1/structureServices/molconvert', loginRoutes.ensureAuthenticated, exports.molConvert
```

Comment:
Why was this removed? Thought this was used by MarvinJS

Reply:
Thanks, I misread how this was being used.  I just added it back.